UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -215,6 +216,11 @@ public void Update()
 			}
 		}
 
+		public Dictionary<Layer, Dictionary<Vector3Int, TileLocation>> GetPresentTiles()
```

Comment:
getters are for java nerds, we use properties instead

Reply:
main reason is because, it's high-risk item no random should be accessing it couldn't think of a better  geter

Reply:
```cs
public Dictionary<Layer, Dictionary<Vector3Int, TileLocation>> GetPreesentTiles => PresentTiles;
```
How is the property less secure?

Reply:
Normally I wouldn't call it ``GetPresentTiles`` but ``PresentTiles`` is already taken.

Reply:
ok